src/index.js

```diff
@@ -35,6 +36,7 @@ module.exports = {
     }
   },
   getWixModulesList: () => wixModulesNames,
+  getComponentsEventsHandlers: () => eventHandlers,
```

Comment:
I actually just checked and when we dynamically import `corvid-types` we are fetching the whole 5MB with the fullCorvidTypes file.
`corvid-types` doesn't use dynamic imports.
So I'm thinking of just importing the `eventHandlers.json` directly from unpkg.
WDYT?

Reply:
I changed all the names to use `event-handlers`

Reply:
i think it's better to have an API that gets a specific component type and decides internally what should be done. it can fetch the full json file now and once it becomes too big (maybe it already is), just fetch a specific component.